extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisorSpec.java

```diff
@@ -52,6 +54,7 @@ public KafkaSupervisorSpec(
       @JsonProperty("dataSchema") DataSchema dataSchema,
       @JsonProperty("tuningConfig") KafkaTuningConfig tuningConfig,
       @JsonProperty("ioConfig") KafkaSupervisorIOConfig ioConfig,
+      @JsonProperty("context") Map<String, Object> context,
```

Comment:
not sure if we should have docs for it because for existing tasks as well context is not mentioned anywhere in the docs. Can add if required.